src/components/List/List.tsx

```diff
@@ -183,7 +185,7 @@ export class List<T = unknown> extends React.Component<ListProps<T>, ListState<T
         const {items, activeItem} = this.state;
         const item = items[index];
         const sortable = this.props.sortable && items.length > 1 && !this.getFilter();
-        const active = index === activeItem || index === this.props.activeItemIndex;
+        const active = index === activeItem;
```

Comment:
`activeItemIndex` prop allows you to fix the active element, it is needed, for example, if there is a dropdown menu with actions for each element and it is not visible for inactive elements. When the dropdown is opened, the active element should be fixed.

Reply:
why would something with "active" in name freeze an element?
and it doesn't even relates to activeItem, which can still be moved